bonus_guides/G_static_assets.md

```diff
@@ -148,6 +148,10 @@ Reload the page. Open the JS console and type `meaning_of_life`. This will retur
 
 Important detail: according to the default configuration there is no ES6 support for files in `web/static/vendor`. Should we need to enable it, look for `plugins: { babel: { ignore:` in `brunch-config.js`.
 
+#### JavaScript Libraries
+
+If you want to use a JS library like jQuery, underscore or whatever you need, you could copy it to the `web/static/vendor` folder, as mentioned above. It may be a little bit easier to use `npm` to install it. Simply add `"jquery": ">= 2.1"` to the dependencies in the `package.json` file in your projects root and run `npm install --save`. Now you can `import $ from "jquery"` in your `app.js`.
```

Comment:
One thing we try to do in the guides is minimize the use of "you" and "your". We tend to substitute "we" and "our" or remove the possessive pronoun altogether. The idea is to foster a subtle sense of inclusion with the reader, as if to say  that we're all in this programming thing together. :)

There are a number of instances of "you" and "your" in this paragraph. Let me take the first sentence or two as an example of how to turn this around:

We may need to use a JavaScript library like jQuery or underscore in our application. As we mentioned above, we could copy the libraries into `web/static/vendor`.